include/swift/AST/DiagnosticsSema.def

```diff
@@ -1354,8 +1354,8 @@ WARNING(pattern_type_not_usable_from_inline_warn,none,
         "%select{%select{variable|constant}0|property}1 "
         "should be '@usableFromInline' or public",
         (bool, bool))
-ERROR(pattern_type_not_usable_from_inline_fixed_layout,none,
-      "type referenced from a stored property in a '@_fixed_layout' struct must "
+ERROR(pattern_type_not_usable_from_inline_frozen,none,
```

Comment:
Do we use this diagnostic for classes too? If so it needs a %choice

Reply:
I bet we do, but it also says "struct" in the message, so…

Reply:
Yeah it was always struct-specific even if it applies to both. I'll happily correct it but I'd like to do that in a follow-on PR than lump it in here.